astroid/raw_building.py

```diff
@@ -424,8 +421,8 @@ class InspectBuilder:
 
     bootstrapped: bool = False
 
-    def __init__(self, manager_instance: AstroidManager | None = None) -> None:
-        self._manager = manager_instance or AstroidManager()
+    def __init__(self, manager_instance) -> None:
```

Comment:
```suggestion
    def __init__(self, manager_instance: AstroidManager) -> None:
```
Unleess there's a reason to not type I don't understand?